tcms_django_plugin/result.py

```diff
@@ -116,11 +116,22 @@ def addError(self, test, err):
         super().addError(test, err)
```

Comment:
This class still needs a prefix attribute because the way CI executes the tests results in 2 TRs both with the same name.

See:
https://tcms.kiwitcms.org/runs/678/
https://tcms.kiwitcms.org/runs/679/

Use "[DJANGO --debug-sql] " for the new prefix.